src/components/Connection.tsx

```diff
@@ -38,6 +38,7 @@ import {
   PopoverContent,
   PopoverTrigger,
 } from "@/components/ui/popover";
+import { ArrowUp, Trash2, Download } from "lucide-react";
```

Comment:
We have already imported some icons from lucide-react, you should add these three icons in that same import.